sphinx/ext/graphviz.py

```diff
@@ -385,7 +385,7 @@ def man_visit_graphviz(self: ManualPageTranslator, node: graphviz) -> None:
 
 
 def on_build_finished(app: Sphinx, exc: Exception) -> None:
-    if exc is None:
+    if exc is None and hasattr(app.builder, 'add_css_file'):
```

Comment:
Could you check `app.builder.format == 'html'` instead here? I think it is a more natural way.

Reply:
Updated as suggested, and tested - works for me.